tests/RKATest/SessionTest.php

```diff
@@ -76,7 +77,9 @@ public function testDestroy()
     {
         @session_start();
         $this->assertEquals(PHP_SESSION_ACTIVE, session_status());
-        @Session::destroy(); // silence headers already sent warning
+        $session = new Session;
+        $session->destroy();
+        // silence headers already sent warning
```

Comment:
I suspect we should move this comment to above `@session_start()` or just remove it.

Reply:
This was fixed later, but should be part of the PR.

https://github.com/akrabat/rka-slim-session-middleware/pull/20/files#diff-9f266881fe264e73745b1641b8af3ea8c6df044d2ebf3679c8c941a1f47d35d0R80